docs/sources/query/_index.md

```diff
@@ -1,347 +1,132 @@
-If the `bool` modifier is provided, vector elements that would have been dropped instead have the value 0 and vector elements that would be kept have the value 1, with the grouping labels again becoming the output label set.
+The following operators can be used in the log stream selector:
 
-Return the streams matching `app=foo` without app labels that have higher counts within the last minute than their counterparts matching `app=bar` without app labels:
+- `=` labels that are exactly equal to the log stream selector
+- `!=` labels that are not equal to the log stream selector
+- `=~` labels that are a regex-match to the log stream selector
+- `!~` labels that are not a regex-match to the log stream selector
 
-```logql
-sum without(app) (count_over_time({app="foo"}[1m])) > sum without(app) (count_over_time({app="bar"}[1m]))
-```
+where the `~` character is used for regular expressions.
 
-Same as above, but vectors have their values set to `1` if they pass the comparison or `0` if they fail/would otherwise have been filtered out:
+### Log pipeline
 
-```logql
-sum without(app) (count_over_time({app="foo"}[1m])) > bool sum without(app) (count_over_time({app="bar"}[1m]))
-```
+The optional log pipline may contain any combination of the following expressions:
+- filters
+- parsers
+- formatters
 
-### Pattern match filter operators
+#### Filter expressions
 
-- `|>` (line match pattern)
-- `!>` (line match not pattern)
+Filter expressions are used to look for a certain string or range of values within the log line or label.
 
-Pattern Filter not only enhances efficiency but also simplifies the process of writing LogQL queries. By eliminating the need for complex regex patterns, users can create queries using a more intuitive syntax, reducing the cognitive load and potential for errors.
+For label filters, the following operators can be used:
+- `==` or `=` for equality.
+- `!=` for inequality.
+- `>` and `>=` for greater than and greater than or equal.
+- `<` and `<=` for lesser than and lesser than or equal.
 
-Within the pattern syntax the `<_>` serves as a wildcard, representing any arbitrary text. This allows the query to match log lines where the specified pattern occurs, such as log lines containing static content, with variable content in between.
+`and` and `or` can also be used to chain operations for label filters.
 
-Line match pattern example:
+For line filters, the following operators can be used:
+- `|=`: Log line contains string
+- `!=`: Log line does not contain string
+- `|~`: Log line contains a match to the regular expression
+- `!~`: Log line does not contain a match to the regular expression
 
-```logql
-{service_name=`distributor`} |> `<_> caller=http.go:194 level=debug <_> msg="POST /push.v1.PusherService/Push <_>`
-```
+#### Parser expressions
 
-Line match not pattern example:
+Parser expressions are used to easily filter logs that are in a standardized format, such as:
+- JSON
+- logfmt
+- Regular Expressions
+- unpack
```

Comment:
Did you leave out [pattern](https://grafana.com/docs/loki/latest/query/log_queries/#pattern) for a specific reason?

Also, we could link to the sections in the Log Queries topic for each parser here.